src/components/CartPage/CartPage.js

```diff
@@ -97,7 +104,13 @@ const CartPage = (props) => {
             <p>{`$${deliveryCost}`}</p>
             <p>{`$${voucher}`}</p>
             <hr />
-            <p>{`$${countTotal + deliveryCost - voucher}`}</p>
+            <input
+              className="total-price"
+              value={`$${calculateTotal}`}
+              type="text"
+              onChange={getPrice(calculateTotal)}
+              readOnly
+            />
```

Comment:
Ok, so you want to show the user its total price. You actually don't need input or onChange event either. Just make all calculations before render and store this data in the state of a component. So if value changes you just trigger simple rerender and display new value.